Doc/library/unittest.mock.rst

```diff
@@ -1308,9 +1308,31 @@ patch.dict
     :func:`patch.dict` can also be called with arbitrary keyword arguments to set
     values in the dictionary.
 
-    :func:`patch.dict` can be used as a context manager, decorator or class
-    decorator. When used as a class decorator :func:`patch.dict` honours
-    ``patch.TEST_PREFIX`` for choosing which methods to wrap.
+:func:`patch.dict` can be used as a context manager, decorator or class
+decorator:
+
+    >>> import patch
```

Comment:
You don't need to `import patch` (which would be `from unittest.mock import patch`), because it's already imported in the `testsetup` (top of the file). Currently it breaks the doctests.

Reply:
Hello @JulienPalard I put this from the coment of @berkerpeksag https://github.com/python/cpython/pull/11456/files#r246624984.

But yes, this is broken the test. I'll remove